proto/hummock.proto

```diff
@@ -64,6 +64,7 @@ message GroupConstruct {
   repeated uint32 table_ids = 3;
   uint64 group_id = 4;
   uint64 new_sst_start_id = 5;
+  bool no_trivial_split = 6;
```

Comment:
Wouldn't it be better to use enum to deal with possible future upgrades?

Reply:
Good suggestion.